src/common/utils.cpp

```diff
@@ -22,6 +22,18 @@ void initExternalModules()
     openpower_guard::libguard::libguard_init(false);
 }
 
+bool isNotStateHostService(std::string firstString)
```

Comment:
nit: 

Commit message header:

`utils: Allow exceptional services that can be hosted as multiple instances`

Reply:
But as part of this commit as I only added  the "chassis" check. So I believe better to leave the commit msg as it is?